sopwith3.txt

```diff
@@ -1,8 +1,8 @@
 [Section]
-Name = Maelstorm
-Version = 3.0.6
+Name = Sopwith 3 (2003)
+Version = 3.0.0
 License = GPL
-Description = Maelstrom is a GPL'd port of the original shareware game for the Macintosh. It is a fast-action, high resolution (640x480) asteroids-like game, with detailed graphics and original sounds.
+Description = Sopwith is a side-scrolling arcade shooter where you fly a biplane (the eponymous Sopwith Camel) and try to wreak havoc on the opponent's bases, ground and air forces.
 Category = 4
```

Comment:
Thanks! Please also add missing Category to the `maelstorm.txt` file.

Reply:
Added missing category.